website/docs/cdktf/debugging.mdx

```diff
@@ -0,0 +1,59 @@
+---
+page_title: Debugging
+description: >-
+  Learn how to troubleshoot problems in your CDKTF application.
+---
+
+# Debugging
+
+## Increasing the verbosity
+
+To get more information about what CDKTF is doing and what problems have occured by setting the `CDKTF_LOG_LEVEL` environment variable to `DEBUG`. If you want to share the logs you can set `CDKTF_LOG_FILE_DIRECTORY` to a file path, a `cdktf.log` file with all logs will be written.
```

Comment:
Grammar seems a bit off here "to get more information ... by setting"

Reply:
Also as the logs are not shared automatically, maybe change that part to "if you want the logs written to a file, ..."